### PR TITLE
CLI- Instructions and aws provider fix

### PR DESCRIPTION
**Pull request by Nilanshrajput**
## Description
Fix for a bug in CLI AWS Provider and updated instruction for deployment using CLI.
Added Terraform installation instructions in readme

## Affected Dependencies
None

## How has this been tested?
Not tested
## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by Nilanshrajput** [comment]
@AmrMKayid  can you review this.

**Comment by Nilanshrajput** [comment]
@AmrMKayid  also it seems like deployment using CLI is not tested as of now,  I am facing some issues deploying on AWS using cli other than this minor bug.

**Comment by IonesioJunior** [comment]
@AmrMKayid , @sachin-101 Could you guys check this? :)

File: README.md [hunk 1]
```diff
@@ -183,8 +183,14 @@ pip install -e .
 
 ### Running CLI
 
+#### Install Terraform
+Check Instructions here : https://learn.hashicorp.com/tutorials/terraform/install-cli#:~:text=If%20you%20get%20an%20error,directory%20where%20Terraform%20was%20installed.
```
Comment (AmrMKayid) [inline README.md:187]: Can you replace it with this link please: 
https://learn.hashicorp.com/tutorials/terraform/install-cli
```suggestion
Check Instructions here: https://learn.hashicorp.com/tutorials/terraform/install-cli
```

File: README.md [hunk 2]
```diff
@@ -183,8 +183,14 @@ pip install -e .
...
+
+
 #### Deploy a Node to AWS
 
+#### Run Terraform init : terraform init
```
Comment (AmrMKayid) [inline README.md:192]: No need for `terraform init` since the CLI already do that once it received the inputs from the user

**Review by AmrMKayid — CHANGES_REQUESTED** [review]
Thanks for the PR and fixes, please check my comments and update the minor changes. :)

File: apps/cli/providers/aws.py [hunk 3]
```diff
@@ -14,12 +14,12 @@ def __init__(self) -> None:
         self.client = boto3.client("ec2")
 
     def regions_list(self):
-        return [region["RegionName"] for region in EC2.describe_regions()["Regions"]]
+        return [region["RegionName"] for region self.client.describe_regions()["Regions"]]
```
Comment (AmrMKayid) [inline apps/cli/providers/aws.py:17]: there is a missing `in` here
```suggestion
        return [region["RegionName"] for region in self.client.describe_regions()["Regions"]]
```

**Review by AmrMKayid — CHANGES_REQUESTED** [review]

**Review by AmrMKayid — APPROVED** [review]
LGTM! Thanks @Nilanshrajput for the fixes!

**Pull request merged**